### ISSUE TITLE
Try to use the code of Appearance Flow in synsin

### ISSUE THREAD

**Issue by zhuge2016** [facebookresearch/synsin#36]
i want use the code of appearance flow prediction;the result of my training is so bad,i want ask that the training setting ,the parameter is look like?
this is my training setting,for  the gpu setting my batchsize is only 8:
python train.py --batch-size 8 --folder 'temp' --num_workers 2  \
         --resume --dataset 'kitti' --use_inv_z --use_inverse_depth --accumulation 'alphacomposite' \
         --model_type 'viewappearance' --refine_model_type 'resnet_256W8UpDown64'  \
         --norm_G 'sync:spectral_batch' --gpu_ids 0,1 --render_ids 0 \
         --suffix '' --normalize_image --lr 0.000025 --log-dir="/home/zhuge/try/sin1/store_old_model/%s"\
my traing result:
ssim : 0.2528 |  psnr : 2.3678 |  Perceptual : 0.8591 |  Total Loss : 8.8668 |  L1 : 0.2760 |
so  what kind of setting should i set?

**Comment by liustu** [comment]
Hi, do you download the dataset of the synsin, if yes, would you like to share it with me? I have worked days for downloading the images of synsin but fails, thank you.
